nmigen/build/tools.py

```diff
@@ -0,0 +1,45 @@
+__all__ = ["yosys_id_escape"]
+
+
+# Implementation of the Yosys RTLIL::IdString escaping function, as defined in
+# https://github.com/YosysHQ/yosys/blob/master/backends/verilog/verilog_backend.cc#L101-L159
+def yosys_id_escape(s):
```

Comment:
This list doesn't have anything to do with Yosys itself, it is a list of (System)Verilog keywords.